src/xrGame/MainMenu.cpp

```diff
@@ -203,6 +204,7 @@ void CMainMenu::Activate(bool bActivate)
 
     if (bActivate)
     {
+        g_DiscordRPC->Init();
```

Comment:
Does it mean we initialize RPC every time we open main menu (well, when menu is active)?

Reply:
Yes. And it is totally wrong. Please, call this function in the constructor of CGamePersistent class, right before Ansel initialization.